bin/cli.js

```diff
@@ -8,13 +8,21 @@ const common = require('metarhia-common');
 
 const metaschema = require('..');
 
-const cwd = process.cwd();
-const apiFile = path.resolve(cwd, process.argv[2]);
+const apiFile = path.resolve(process.cwd(), process.argv[2]);
```

Comment:
Wouldn't this be affected by the same issue as in browser-runner? (Shouldn't we search for project root dir?)